services/mail.ts

```diff
@@ -39,6 +39,13 @@ export class Mail {
     return file
   }
 
+  public getTimeZone(date: date) {
+    let getDateObject = new Date(date)
+    let formatDate = Intl.DateTimeFormat(getDateObject)
```

Comment:
`Intl.DateTimeFormat` expects `string | string[] | undefined` as argument. Passing a `Date` object is a type error and will fail the build. 

Run `npx tsc` to check if the build passes, we should probably add that the CI checks

Reply:
We're already running tsc on the CI.